### PR TITLE
Features/ create payment logic for appointment

### PR DESCRIPTION
**Pull request by ProggerOleg**
## Describe your changes

- I made cron operation to run every 10 minutes and check whether there are unpaid, but accepted by seeker appointments
- Created logic to check type of appointment, if it's one time it is charged once, else it's charged every week since first day from list of days when appointment happens.
- It's only charged for those appointments, which were accepted by seeker, if seeker reject's appointment it won't be charged.
- Also moved payForHourOfWork, implemented it's logic: if it's first payment for recurring appointment, it will be charged -1 hour because we already charged it when user have chosen caregiver. If it's one-time appointment, it will be charged - 1 hour by default.
- I created endpoint to accept and reject activity log
- Made balance float number
## Issue ticket code (and/or) and link
- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/CC-215)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [x] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Backend

- [x] Swagger documentation updated
- [x] Database requests are optimized and not redundant
- [ ] Unit tests written
- [x] use ConfigService instead of process.env
- [x] use transactions if there is a call chain that mutates data in different tables
- [ ] use @index decorator for frequently requested data

### CODE REVIEW

File: src/modules/caregiver-info/caregiver-info.controller.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class CaregiverInfoController {
   }
 
   @Get(CaregiverApiPath.DetailedInfo)
-  @ApiOperation({ summary: 'Get detailed caregiver information' })
+  @ApiOperation({ summary: 'Get filtred caregivers list' })
```
Comment (nadiia-tsytsylina) [inline src/modules/caregiver-info/caregiver-info.controller.ts:71]: Why did you change it back? it should be Get detailed caregiver information in description
image

Comment (nadiia-tsytsylina) [reply src/modules/caregiver-info/caregiver-info.controller.ts:71]: also I think, you had some problems with solving conflicts, because you deleted all files, that I created in my last PR
image

Comment (ProggerOleg) [reply src/modules/caregiver-info/caregiver-info.controller.ts:71]: thanks for noticing, you're right
